depends/packages/crate_base64.mk

```diff
@@ -1,9 +1,9 @@
-package=crate_semver
```

Comment:
Yes; because these package definitions are so short and have common content, the percentage of common lines is sufficiently high that git tries to be helpful and treat the independent deletion and addition as a move.